packages/fluxible-addons-react/package.json

```diff
@@ -14,13 +14,12 @@
   },
   "dependencies": {
     "hoist-non-react-statics": "^3.0.1",
-    "inherits": "^2.0.1",
     "prop-types": "^15.5.8"
   },
   "peerDependencies": {
     "fluxible": ">=1.0.0",
     "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0 ||  || ^16.0.0"
+    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0"
```

Comment:
if we are bumping major version, we can remove `^0.14.0` from this line and the line above